src/components/nve-tab-panel/nve-tab-panel.component.ts

```diff
@@ -0,0 +1,62 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import styles from './nve-tab-panel.styles';
+import { watch } from '../../utils/watch';
+import { classMap } from 'lit/directives/class-map.js';
+
+let id = 0;
+
+/**
+ * Innholdet i en fane.
+ *
+ * NveTab er selve knappen for å velge fane
+ * NveTabPanel er innholdet for hver fane
+ * NveTabGroup er den som disse skal ligge i
+ * Disse kobles sammen med `panel` på NveTab som skal være det samme som `name` på NveTabPanel
+ *
+ * @csspart base - Hele komponenten er inne i denne.
+ *
+ */
+
+@customElement('nve-tab-panel')
+export default class NveTabPanel extends LitElement implements INveComponent {
+  @property({ reflect: true, type: String }) testId: string | undefined = undefined;
+  @property({ reflect: true, type: String }) name: string = '';
```

Comment:
Denne må dokumenteres. 
Hva er bakgrunnen for at du har valgt panel som attributt i nve-tab mens tilsvarende attributt i nve-tab-panel heter name? 
Kunne de hett name begge to? Eller kunne name i nve-tab-panel hett tab? Jeg prøver bare å drodle om hva som er mest logisk for "brukeren":)